export/export.go

```diff
@@ -501,3 +515,8 @@ func (s *Service) applyPods(ctx context.Context, r *ResourcesForImport, eg *util
 	}
 	return nil
 }
+
+// The name of PriorityClass that is prefixed with `system-`, is reserved by the system.
+func filterPriorityClass(name string) bool {
```

Comment:
Okay. Rename it to `isSystemPriorityClass`. (IMO, I feel a little strange if`filterXX` returns bool.)